### PR TITLE
feat(maleo-core/render): changed the way we render wrap

### PR DESCRIPTION
**Pull request by alvinkl**
change the structure of rendering wrap, now wrap has container and app component exposed as props.
by having this we will be able to add `<Link />` tag to `Wrap` and have better access to build custom error page

#### Pre-Submission Checklist

- [ ] Your pull request targets the `canary` branch of Maleo.js.
- [x] Branch starts with either `fix/`, `feature/`, etc (e.g. `fix/build-issue`. [more info](https://github.com/airyrooms/maleo.js/blob/canary/CONTRIBUTING.md))
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `yarn test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [x] Optimization (non-breaking change which provides better optimization)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
- [ ] Update Documentation (added new docs or doc updates)
- [ ] Other ( )

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Re #132

---

#### Description

Describe changes 

---

### CODE REVIEW

File: packages/Maleo.js/src/client/client.tsx [hunk 1]
```diff
@@ -19,12 +20,25 @@ export const init = async () => {
...
+    };
+    const containerProps = {};
 
     const RenderApp = () => (
-      <Wrap {...wrapProps}>
-        <App data={data} routes={routes} location={location} {...appProps} {...wrapProps} />
-      </Wrap>
+      <Wrap
+        App={App}
+        Container={ContainerComponent}
```
Comment (AndariasSilvanus) [inline packages/Maleo.js/src/client/client.tsx]: How if we put `Container` before `Wrap` since `Container` isn't extensible to the user?

Comment (alvinkl) [reply packages/Maleo.js/src/client/client.tsx]: I think it is more versatile to have Container exposed to use, the user can choose what to do with the `Container` itself. And they are capable to add some provider above `Container`

Comment (AndariasSilvanus) [reply packages/Maleo.js/src/client/client.tsx]: Ok, I got your point

**Comment by AndariasSilvanus** [comment]
LGTM

**Review by AndariasSilvanus — APPROVED** [review]

**Pull request merged**